spring-boot-admin-server/src/main/java/de/codecentric/boot/admin/server/services/InfoUpdateTrigger.java

```diff
@@ -41,7 +41,8 @@ public class InfoUpdateTrigger extends AbstractEventHandler<InstanceEvent> {
 	public InfoUpdateTrigger(InfoUpdater infoUpdater, Publisher<InstanceEvent> publisher) {
 		super(publisher, InstanceEvent.class);
 		this.infoUpdater = infoUpdater;
-		this.intervalCheck = new IntervalCheck("info", this::updateInfo, Duration.ofMinutes(5), Duration.ofMinutes(1));
+		this.intervalCheck = new IntervalCheck("info", this::updateInfo, Duration.ofMinutes(5), Duration.ofMinutes(1),
+				Duration.ofMinutes(10));
```

Comment:
I don't like that we have defaults here (because we already have them in the AdminServerProperties).
In the AdminServerAutoConfiguration we call the setters to override these values anyways.

Can't we just pass the values to this constructor here? Then we could get rid of these defaults here and also get rid of the setters which are just passing the value to the next object.

Reply:
hm, the setters are also used in tests, makes them easier

Reply:
changed this, but kept the setters for tests